sharding-jdbc/sharding-jdbc-core/src/test/java/org/apache/shardingsphere/shardingjdbc/jdbc/core/statement/ShardingSpherePreparedStatementTest.java

```diff
@@ -126,7 +130,7 @@ public void assertMultiValuesWithGenerateShardingKeyColumn() throws SQLException
             }
         }
     }
-
+    
```

Comment:
Please keep original indent

Reply:
It's wired , there is two kinds of ident, some files no indent ,some files has ident .  which one should I use?

Reply:
we prefer just keep 4 spaces

Reply:
use 4 spaces.